bin/release_automation.sh

```diff
@@ -58,73 +58,76 @@ fi
 
 # Create Git branch
 RELEASE_BRANCH="release/$VERSION_NUMBER"
-git switch -c "$RELEASE_BRANCH" || { echo "Error: could not create '$RELEASE_BRANCH' branch."; exit 1; }
+ohai "Create Git branch '$RELEASE_BRANCH' in gutenberg-mobile."
+execute "git" "switch" "-c" "$RELEASE_BRANCH" 
 
 # Create Git branch in Gutenberg
 GB_RELEASE_BRANCH="rnmobile/release_$VERSION_NUMBER"
+ohai "Create Git branch '$GB_RELEASE_BRANCH' in gutenberg."
 cd gutenberg
-git switch -c "$GB_RELEASE_BRANCH" || { echo "Error: could not create '$GB_RELEASE_BRANCH' branch."; exit 1; }
+execute "git" "switch" "-c" "$GB_RELEASE_BRANCH" 
 cd ..
 
 # Set version numbers
-for file in 'package.json' 'package-lock.json' 'gutenberg/packages/react-native-editor/package.json'; do
-    npx json -I -f "$file" -e "this.version='$VERSION_NUMBER'" || { echo "Error: could not update version in ${file}"; exit 1; }
```

Comment:
We may be able to remove [the json dependency from the package.json file](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/package.json#L42) now.

Reply:
That's great!